doc/cephadm/upgrade.rst

```diff
@@ -188,3 +188,43 @@ you need. For example, the following command upgrades to a development build:
   ceph orch upgrade start --image quay.io/ceph-ci/ceph:recent-git-branch-name
 
 For more information about available container images, see :ref:`containers`.
+
+
+Migrations
+==========
+
+.. _cephadm-migration-nfs:
+
+16.2.7 - NFS export migration
+-----------------------------
+
+This migration should be done fully automated, but in case something goes
+wrong while the upgrade is in progress, there is also a manual procedure
+
+in 16.2.6, the pool for storing NFS ganesha exports was arbitrary, after
+16.2.7, the pool name is always ``.nfs``. to create the Pool, run::
+
+  osd pool create pool .nfs
+  osd pool application enable .nfs nfs
+
+Previously, there was an extra parameter which was the RAODOS namespace. 
+After 16.2.7, the namespace is now always set to the cephadm service id. 
+
+Next, we have to copy over the ``grace`` object from the old location to
```

Comment:
`rados [get|put] --pool <pool> --namespace <namespace>` ...

but I don't think we need todo this since the grace db will be recreated in the new pool during redeploy....